components/brave_new_tab_ui/reducers/new_tab_reducer.ts

```diff
@@ -308,6 +308,11 @@ export const newTabReducer: Reducer<NewTab.State | undefined> = (state: NewTab.S
       }
       break
 
+    case types.ON_BINANCE_USER_TLD:
+      state = { ...state }
+      state.binanceState.userTLD = payload.userTLD
```

Comment:
nit: 

```js
state = { ...state, binanceState: { userTLD: payload.userTLD } }
```

It's better to read to me personally. WDYT?

Reply:
sure, i've changed this